cli/test.js

```diff
@@ -46,6 +50,11 @@ function test(command, argv) {
     process.exit(exitCode);
   };
 
+  if (specsGlob.length === 0) {
+    logger.info('No spec files located. Stopping command from running.');
```

Comment:
I'd maybe change the wording to "No spec files located.  Skipping test command."